pkg/services/rpcsrv/client_test.go

```diff
@@ -2466,6 +2468,43 @@ func TestClient_GetVersion_Hardforks(t *testing.T) {
 	require.InDeltaMapValues(t, expected, v.Protocol.Hardforks, 0)
 }
 
+func TestServer_GetVersion(t *testing.T) {
+	chain, srv, _ := initServerWithInMemoryChain(t)
+
+	resRaw, rpcErr := srv.getVersion(params.Params{})
+	require.Nil(t, rpcErr, "expected no RPC error from getVersion")
+	version, ok := resRaw.(*result.Version)
+	require.True(t, ok, "expected result to be *result.Version")
+
+	require.Equal(t, srv.coreServer.ID(), version.Nonce)
+	require.Equal(t, srv.coreServer.UserAgent, version.UserAgent)
+
+	require.Equal(t, srv.config.MaxIteratorResultItems, version.RPC.MaxIteratorResultItems)
+	require.Equal(t, srv.config.SessionEnabled, version.RPC.SessionEnabled)
+	require.Equal(t, srv.config.SessionExpansionEnabled, version.RPC.SessionExpansionEnabled)
+
+	cfg := chain.GetConfig()
+
+	assert.Equal(t, address.NEO3Prefix, version.Protocol.AddressVersion)
+	require.Equal(t, cfg.Magic, version.Protocol.Network)
+	require.Equal(t, int(chain.GetMillisecondsPerBlock()), version.Protocol.MillisecondsPerBlock)
+	require.Equal(t, chain.GetMaxTraceableBlocks(), version.Protocol.MaxTraceableBlocks)
+	require.Equal(t, chain.GetMaxValidUntilBlockIncrement(), version.Protocol.MaxValidUntilBlockIncrement)
+	require.Equal(t, cfg.MaxTransactionsPerBlock, version.Protocol.MaxTransactionsPerBlock)
+	require.Equal(t, cfg.MemPoolSize, version.Protocol.MemoryPoolMaxTransactions)
+	require.Equal(t, byte(cfg.GetNumOfCNs(chain.BlockHeight())), version.Protocol.ValidatorsCount)
+	require.Equal(t, cfg.InitialGASSupply, version.Protocol.InitialGasDistribution)
+
+	require.Equal(t, cfg.SeedList, version.Protocol.SeedList)
+	require.Equal(t, cfg.CommitteeHistory, version.Protocol.CommitteeHistory)
+	require.Equal(t, cfg.P2PSigExtensions, version.Protocol.P2PSigExtensions)
+	require.Equal(t, cfg.StateRootInHeader, version.Protocol.StateRootInHeader)
+	require.Equal(t, cfg.ValidatorsHistory, version.Protocol.ValidatorsHistory)
+
+	require.Equal(t, cfg.SaveInvocations, version.Application.SaveInvocations)
+	require.Equal(t, cfg.KeepOnlyLatestState, version.Application.KeepOnlyLatestState)
```

Comment:
The point of this test is to check RPC client. So instead of calling RPC server's handler directly, follow the behaviour of `TestClient_GetVersion_Hardforks`: create RPC client, call `c.GetVersion()`, check that these two fields are set.